packages/ui/src/components/va-input/components/VaInputWrapper.vue

```diff
@@ -80,6 +80,7 @@
     </div>
 
     <div class="va-input-wrapper__message-list-wrapper">
+      <slot name="messages" />
```

Comment:
Let user completely redefine messages look. Please, check if VaMessageList is exposed in vuestic-ui.

```suggestion
      <slot name="messages" v-bind="{ messages: messagesComputed, errorLimit, color: messagesColor }">
            <va-message-list
                :color="messagesColor"
                :model-value="messagesComputed"
                :limit="errorLimit"
              />
      </slot>
```